docs/usage/channels.rst

```diff
@@ -11,115 +11,143 @@ Channels provide:
-        managing :term:`subscribers <subscriber>`, reading messages from the
-        :term:`backend`, putting them in the appropriate :term:`event stream`, and
-        publishing data to the backend
+        backend
+            A :class:`ChannelsBackend <.backends.base.ChannelsBackend>`. This object
+            manages communication between the plugin and the :term:`broker`, publishing
+            messages to and receiving messages from it. Each plugin instance is associated
+            with exactly one backend.
 
+        broker
+            Responsible for receiving and publishing messages to all connected
+            :term:`backends <backend>`; All backends sharing the same broker will have
+            access to the same messages, allowing for inter-process communication. This is
+            typically handled by a separate entity like `Redis <https://redis.io/>`_
 
-Flowcharts
-++++++++++
+        plugin
+            The :class:`~.plugin.ChannelsPlugin`, a central instance
+            managing :term:`subscribers <subscriber>`, reading messages from the
+            :term:`backend`, putting them in the appropriate :term:`event stream`, and
+            publishing data to the backend
 
+        channel
+            A named group of subscribers, to which data can be published. Subscribers can
+            subscribe to multiple channels, and channels can have multiple subscribers
 
-.. mermaid::
-    :align: center
-    :caption: Publishing flow from the application to the :term:`broker`
+        subscription
+            A connection between a :term:`subscriber` and a :term:`channel`, allowing the
+            subscriber to receive events from the channel
 
-    flowchart LR
-        Backend(Backend) --> Broker[(Broker)]
+        backpressure
+            A mechanism to prevent the backlog of a :term:`subscriber` from growing
+            indefinitely, by either dropping new messages or evicting old ones
 
-        Plugin{{Plugin}} --> Backend
+        history
+            A set of previously published :term:`events <event>`, stored by the :term:`backend`
+            and available to be pushed to a :term:`subscriber`
 
-        Application[[Application]] --> Plugin
+        fanout
+            The process of sending a message to all subscribers of a channel
 
+        eviction
+            A :term:`backpressure` strategy, dropping the oldest message in the backlog when
+            a new one is added while the backlog is full
 
-.. mermaid::
-    :align: center
-    :caption: Fanout flow of data from the :term:`broker` to the sockets, showing multiple plugin instances
+        backoff
+            A :term:`backpressure` strategy, dropping newly incoming messages as long as the
+            backlog is full
 
-    flowchart TD
-        Broker[(Broker)]
+Flowcharts
+++++++++++
 
-        Broker --> Backend_1(Backend)
-        Broker --> Backend_2(Backend)
+.. dropdown:: Click to expand flowcharts
```

Comment:
Is this actually necessary? We already have the side-nav where you can easily skip the section if you're not interested in that.